src/test/unit/conversion-manifest_test.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license
+ * Copyright (c) 2018 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {assert} from 'chai';
+import {readJson} from 'fs-extra';
+import * as path from 'path';
```

Comment:
I'm starting to get in the habit of importing `*` less.  `import {join} from 'path';`

Just a thought!

Reply:
I know, this is a hard habit for me to break. But I like the signal when I see `path.join` that I can assume it's the `path.join()` method that I already know, without investigating.